examples/Web/README.md

```diff
@@ -0,0 +1,35 @@
+## Web Example
+
+This project demonstrates how to integrate Kafka with a .NET web application. It provides
+examples of both producing and consuming messages.
+
+
+#### KafkaClientHandle
+
+Demonstrates how to wrap the Confluent.Kafka producer in a class that can be registered as
+a singleton service, including how to bind client configuration from an injected IConfiguration
+instance to a Confluent.Kafka.ProducerConfig object.
```

Comment:
I think it should briefly but with force explain why long-running clients are required (we shall not pretend client-per-request to be a viable solution).

Reply:
yes good idea. thanks for pushing me to do this.